cves/kernel/CVE-2014-9730.yml

```diff
@@ -84,14 +95,15 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: e237ec37ec154564f8690c5bd1795339955eeef9
   note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+    Manually confirmed
+- commit: 1da177e4c3f41524e886b7f1b8a0c1fc7321cac2
+  note: This seems to be either a merge commit or a duplicate fix
+- commit: 0e5cc9a40ada6046e6bc3bdfcd0c0d7e4b706b14
+  note: This seems to be either a merge commit or a duplicate fix
+- commit: cb00ea3528eb3c09eae9871d6e7d038776e952e2
```

Comment:
cb00ea3528eb3c09eae9871d6e7d038776e952e2 is just a coding style update, not a fix commit.
1da177e4c3f41524e886b7f1b8a0c1fc7321cac2 looks like it is the initial upload of the code to the repo